tests/unit/createRoute.ts

```diff
@@ -21,15 +19,15 @@ suite('createRoute', () => {
 			}
 		});
 
-		const selections = route.select({} as C, [], false, new UrlSearchParams());
+		const selections = route.select({} as Context, [], false, new UrlSearchParams());
 		assert.lengthOf(selections, 0);
 	});
 
 	test('guard() receives the context', () => {
-		const context: C = {};
-		let received: C = <any> undefined;
+		const context: Context = {};
+		let received: Context = <any> undefined;
```

Comment:
curious, did we say we were going to standardise adding a trailing `!` to prevent the null check in tests over adding a cast to `<any>`?